src/Controller/FindByPid/GalleryController.php

```diff
@@ -41,7 +42,8 @@ public function __invoke(
             'network' => $network,
             'galleries' => $galleries,
             'brand' => $brand,
-        ]);
+            'isGallery' => $isGallery,
+         ]);
```

Comment:
Think there's an extra space here
```suggestion
        ]);
```